_posts/2018-01-30-release-3.2.0.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Release: 3.2.0"
+date:   2018-01-30
+---
+
+We are pleased to announce the release of OMERO.figure 3.2.0.
+
+New features in OMERO.figure 3.2.0:
+
+ - Page layout supports cropping of the page to fit around image panels ([PR #252](https://github.com/ome/omero-figure/pull/252)).
+ - Creation of new labels from Tags on the image ([PR #254](https://github.com/ome/omero-figure/pull/254)).
+ - Allow renaming of saved figures ([PR #251](https://github.com/ome/omero-figure/pull/251)).
+ - Rename of channel rendering setting "Reverse intensity" to "Inverted" ([PR #238](https://github.com/ome/omero-figure/pull/238)).
+
+These features are included in the updated <a href="http://help.openmicroscopy.org/figure.html">User Manual</a>
+and a new <a href="https://www.youtube.com/watch?v=mjlNBUoGKRc&">demo video</a>.
```

Comment:
One tiny nitpick, that you don't need the '&' at the end of the demo video URL, otherwise I think this is all good now thanks